test/filesystemevent/main.d

```diff
@@ -48,135 +48,6 @@ import ocean.task.Task;
 
 import ocean.task.Scheduler;
 
-deprecated class FileModificationTestTask: Task
```

Comment:
I recommended @leandro-lucarella-sociomantic to nother bother and simply delete these one as it is rather old and going away soon'ish :) Don't mind either way tbh, just didn't seem important.